packages/core/test/test-util/test-services/openapi/test-service/entity-api.ts

```diff
@@ -4,68 +4,57 @@
  * This is a generated file powered by the SAP Cloud SDK for JavaScript.
  */
 import { OpenApiRequestBuilder } from '../../../../../src';
-import { EntityApi } from './openapi/api';
-import { TestEntity } from './openapi/model';
+import { TestEntity } from './model';
 
-export const TestServiceEntityApi = {
-  getAllEntities: (args?: {
-    stringParameter?: string,
-    integerParameter?: number,
-    dollarParameter?: string,
-    dotParameter?: string,
-    enumStringParameter?: 'value1' | 'value2',
-    enumInt32Parameter?: 1 | 2,
-    enumDoubleParameter?: 1 | 2,
-    enumBooleanParameter?: boolean
-  }) => new OpenApiRequestBuilder<EntityApi, 'getAllEntities'>(
-    EntityApi,
-    'getAllEntities',
-    args?.stringParameter,
-    args?.integerParameter,
-    args?.dollarParameter,
-    args?.dotParameter,
-    args?.enumStringParameter,
-    args?.enumInt32Parameter,
-    args?.enumDoubleParameter,
-    args?.enumBooleanParameter
+export const EntityApi = {
+  getAllEntities: (queryParameters?: {'stringParameter'?: string,
+  'integerParameter'?: number,
+  '$dollarParameter'?: string,
+  'dot.parameter'?: string,
+  'enumStringParameter'?: 'value1' | 'value2',
+  'enumInt32Parameter'?: 1 | 2,
+  'enumDoubleParameter'?: 1 | 2,
+  'enumBooleanParameter'?: true | false}) => new OpenApiRequestBuilder<TestEntity[]>(
+    'get',
+    '/entities',
+    {
+          queryParameters
+        }
   ),
-  updateEntityWithPut: (args?: {
-    body?: TestEntity[]
-  }) => new OpenApiRequestBuilder<EntityApi, 'updateEntityWithPut'>(
-    EntityApi,
-    'updateEntityWithPut',
-    args?.body
+  updateEntityWithPut: (body: TestEntity[] | undefined) => new OpenApiRequestBuilder<any>(
```

Comment:
That was not my understanding. Can you elaborate why?
If there is a known return type we pass this, if not we use any. The only question where I wasn't sure after our discussion is what if there is 'application/json' and other media types.

Reply:
@FrankEssenberger @jjtang1985 
So we talked about some, but not all cases for the response / request body types:
* only 'application/json' => return type based on schema
* only unknown type, e. g. 'application/xml' => return type any
* multiple types including 'application/json' => ??

Reply:
The fact is that not all the specs provide enough info about the media type as a contract. 
E.g., the WF does not provide any info for return type.

- I would return `any` in case there is not clear info, because it's really `any`. 
- We should only give a type, once the spec gives clear information and/or users provide some input based on their knowledge.

So, without supporting the user import, the user can still cast the response. As sdk, to be safe, I would only give you the type, when only `application/json` is mentioned. For the rest, return `any`. Maybe even for this PR, just return any.

Reply:
This is already implemented as you described 😊 In case there is application/json and others I return `ParsedType | any` which is any, but it is nice to have for reference.